5.0/en/0x14-V6-Cryptography.md

```diff
@@ -20,7 +20,10 @@ Applications need to be designed with strong cryptographic architecture to prote
 
 Architectural requirements are intrinsic to the entire code base, and thus difficult to unit or integration test. Architectural requirements require consideration in coding standards, throughout the coding phase, and should be reviewed during security architecture, peer or code reviews, or retrospectives.
 
-It is also important to ensure that all cryptographic assets, such as algorithms, keys, and certificates, are regularly discovered, inventoried, and assessed.
+It is also important to ensure that all cryptographic assets, such as algorithms, keys, and certificates, are regularly discovered, inventoried, and assessed. For Level 3, this should include the use of static and dynamic scanning to discover the use of cryptography in an application. Tools such as SAST and DAST may help with this but it is possible that dedicated would be needed for this. Freeware examples of this include:
```

Comment:
I think a word is missing after "dedicated". "dedicated tooling"?

Reply:
Clarified the wording:
https://github.com/OWASP/ASVS/pull/2450/commits/548e9c70b7fda2fc97e400d5919d66341df93c81